internal/services/mongocluster/mongo_cluster_resource_test.go

```diff
@@ -36,9 +36,12 @@ func testAccMongoCluster_basic(t *testing.T) {
 			Config: r.basic(data),
 			Check: acceptance.ComposeTestCheckFunc(
 				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("connection_strings.0.connection_string").HasValue(
+					fmt.Sprintf(`mongodb+srv://adminTerraform:QAZwsx123basic@acctest-mc%d.mongocluster.cosmos.azure.com/?tls=true&authMechanism=SCRAM-SHA-256&retrywrites=false&maxIdleTimeMS=120000`,
+						data.RandomInteger)),
 			),
 		},
-		data.ImportStep("administrator_password", "create_mode"),
+		data.ImportStep("administrator_password", "create_mode", "connection_strings.0.connection_string"),
```

Comment:
We'll need to add `"connection_strings.0.connection_string"` to all data.ImportSteps in this test file

Reply:
Done